src/Prompt.js

```diff
@@ -85,10 +60,8 @@ class Prompt extends React.Component {
           :
```

Comment:
You can simplify this `p` tag a lot by simply evaluating 
```
 { smileyDescription || chosenSmiley || `empty` }
```
i.e. if the user is hovering, it will show the description, if not, their chosen smiley, if not, the `empty` string which will be hidden.